docs/sdks/stash/uid2-sdk-ref-mobile-ima.md

```diff
@@ -0,0 +1,113 @@
+# Developer Guide for UID2 IMA Plugin (iOS and Android)
```

Comment:
Please disable this page for now - Google hasn't fixed their problems on their end so IMA Plugin isnt working and can't be used. The details below are not up to date as well. When we hear from google then we can re-enable this page and edit it and publish it properly.

Reply:
Yes... sorry. I should have removed these files before asking you to review. Stash folder and Word folder were not intended for review.